platform/darwin/docs/theme/assets/css/jazzy.css.scss

```diff
@@ -192,29 +193,33 @@ pre {
   margin: 0 0 1.275em;
   padding: .85em 1em;
   overflow: auto;
-  background: $code_bg_color;
+  background: $color_light;
   font-family: $code_font;
   border-radius: 4px;
 }
 
 code {
   font-family: $code_font;
-  font-size: .85em;
+  font-size: 13px;
   line-height: .85em;
 }
 
 p, li {
   > code {
-    background: $code_bg_color;
-    padding: .2em;
+    background: $color_light;
+    padding: 0.1em 0.2em;
+    font-weight: bold;
+    border: 1px solid #D5D5D5;
```

Comment:
Toning it down a bit would probably be fine, but I do like that there’s more emphasis on the inline code blocks now.